client/components/async-load/index.jsx

```diff
@@ -6,8 +6,13 @@ import { omit } from 'lodash';
 
 export default class AsyncLoad extends Component {
 	static propTypes = {
+		noPlaceholder: PropTypes.bool,
+		placeholder: PropTypes.node,
 		require: PropTypes.func.isRequired,
-		placeholder: PropTypes.node
+	};
+
+	static defaultProps = {
+		noPlaceholder: false,
```

Comment:
couldn't we detect a `null` on `placeholder` and skip the boolean flag?

what should happen if we pass a valid placeholder _and_ `noPlaceholder: true`?